### DISCUSSION TITLE
IP Address

### DISCUSSION THREAD

**Discussion by JasonMM2000** [vwt12eh8/hassio-ecoflow#90] [General]
I am looking for the IP address of the ecoflow delta pro.  Can anyone please share the format of the hostname or the beginning of the MAC address to help find it?  

Thank you

**Comment by JasonMM2000** [comment]
I finally found it.  In my Asus router it is listed as:

Device Type: Espressif Inc.   
Client Name: LWIP
MAC starts with: EC:94:CB

Hopefully this helps someone else.

**Reply by yreiter** [reply]
have the same question for River 2 Max. can't find it

**Reply by keithcroshaw** [reply]
@JasonMM2000 it did help, I've been trying to identify all IOT devices on my network and I couldn't figure out what my LWIP device was. If it's espressif I'm shocked my River G1 doesn't have bluetooth. It's probably just disabled and they don't want to give away a feature for free.
